flink-streaming-java/src/test/java/org/apache/flink/streaming/util/MockStreamConfig.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.streaming.util;
+
+import org.apache.flink.api.common.typeutils.base.IntSerializer;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.runtime.jobgraph.OperatorID;
+import org.apache.flink.streaming.api.graph.StreamConfig;
+
+/**
+ * Handy mock for {@link StreamConfig}.
+ */
+public class MockStreamConfig extends StreamConfig {
+	public MockStreamConfig() {
+		super(new Configuration());
+
+		setTypeSerializerIn1(IntSerializer.INSTANCE);
```

Comment:
not sure about this one. I would expect a `MockStreamConfig` to be generally usable, but this one can mess things up in regards to serializers. I'd just remove the mocking and replace it with 
```
StreamConfig streamConfig = new StreamConfig();
streamConfig.setTypeSerializerIn1(IntSerializer.INSTANCE);
```

Reply:
Good point, fixed.